sphinxcontrib/issuetracker/resolvers.py

```diff
@@ -200,6 +202,18 @@ def lookup_jira_issue(app, tracker_config, issue_id):
         return Issue(id=issue_id, title=title, closed=closed, url=url)
 
 
+def lookup_pagure_issue(app, tracker_config, issue_id):
+    if not tracker_config.url:
+        raise ValueError('URL required, try: https://pagure.io/')
+    response = requests.get(PAGURE_API_URL.format(tracker_config, issue_id),
+                            headers={'Accept': 'application/json'})
+    response.raise_for_status()
```

Comment:
Hmm on second thought I guess this should just be a warning, rather than crashing the whole sphinx-build. In case you are building the docs without network access, like in Koji.